core/app/views/refinery/admin/_tmp_search.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="search">
+  <form method='GET' action='<%= url %>'>
+    <div class="search-form">
+      <%= text_field :search, nil, :id => 'search', :type => 'search', :name => 'search', :value => local_assigns[:search],
+                     :class => 'form-control',
+                     :placeholder => t('search_input_notice', :scope => 'refinery.admin.search')
+      %>
+      <%= hidden_field :dialog, nil, :name => 'dialog', :id => nil, :value => true if from_dialog? %>
+      <%= hidden_field :callback, nil, :name => 'callback', :id => nil, :value => @callback if @callback.presence %>
+      <span id="refinery-search-btn" class="search-btn">
+        <button type="submit"><i class="icon icon-search"></i>
+          <div class="inline loader xs bump-up"></div>
+        </button>
+      </span>
+    </div><!-- /input-group -->
+  </form>
+</div>
```

Comment:
I'm not in favor of a temporary partial name like `_tmp_search.html.erb` and personaly i use Elasticsearch in frontend and probably soon in backend. So i don't know if this partial should be in core or in an extension.

Any thoughts @parndt ?